src/soilbiogeochem/SoilBiogeochemNitrogenFluxType.F90

```diff
@@ -340,7 +340,7 @@ subroutine InitHistory(this, bounds)
           data1dptr => this%decomp_cascade_sminn_flux_col(:,l)
           if ( decomp_cascade_con%cascade_receiver_pool(l) /= 0 ) then
              fieldname = 'SMINN_TO_'//&
-                  trim(decomp_cascade_con%decomp_pool_name_history(decomp_cascade_con%cascade_receiver_pool(l)))//'N_'//&
+                  trim(decomp_cascade_con%decomp_pool_name_short(decomp_cascade_con%cascade_receiver_pool(l)))//'N_'//&
```

Comment:
Here and in one other place I replaced `decomp_pool_name_history` with `decomp_pool_name_short` because I got an error as a result of a fieldname that was too long.

The longname still contains all the necessary information for users to decipher which variable they are looking at.

Reply:
Yes, this makes sense.